src/components/Scribe/types.ts

```
@@ -0,0 +1,61 @@
+        comparer: (a: T[number], b: T[number]) => boolean,
+        updatableFields: (keyof T[number])[]
+        onDelete: (fullItem: T[number]) => Promise<unknown> | unknown;
+        onAdd: (fullItem: T[number]) => Promise<unknown> | unknown;
+        onUpdate: (strippedItem: Partial<T[number]>, fullItem: T[number]) => Promise<unknown> | unknown;
+    } : { onUpdate: (strippedItem: Partial<T>, fullItem: T) => Promise<unknown> | unknown; })
+
+export type ScribeInputProps<T> = {
+    children: (props: { value: () => T, aiResponse?: T } & (T extends any[] ? { actions?: ScribeActions<T> } : {})) => React.ReactNode;
+} & ScribeInput<T>
+
+export interface ScribeControlProps { }
```

Biome

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)